setup.cfg

```diff
@@ -30,8 +30,10 @@ extend-ignore =
     E203, # Incompatible with black see https://github.com/psf/black/issues/315
     W503, # Incompatible with black
     B901 # Combine yield and return statements in one function
-max-complexity = 83
-max-line-length = 138
+max-complexity = 36
+max-line-length = 120
+per-file-ignores =
+   astroid/rebuilder.py: C901
```

Comment:
Why not do this with a in-line comment: `# noqa: C901` instead of ignoring the whole file?
Doing so makes any editor of that file think about decreasing, not increasing complexity.

Reply:
Yeah I think `in-line` makes sense.